Documentation/advanced-configuration.md

```diff
@@ -0,0 +1,318 @@
+# Advanced Cluster Configuration
+
+These examples show how to perform advanced configuration tasks on your Rook storage cluster.
+
+## Prerequisites
+Most of the examples make use of the `ceph` client command.  A quick way to use the Ceph client suite is from a [Rook Toolbox](https://github.com/rook/rook/blob/master/Documentation/toolbox.md) container.
+
+The Kubernetes based examples assume Rook OSD pods are in the `rook` namespace.  If you run them in a different namespace, modify `kubectl -n rook [...]` to fit your situation.
+
+## Reweight your OSDs
+If your storage nodes have varying sizes of storage drives, you may benefit from changing their [CRUSH Map](http://docs.ceph.com/docs/kraken/rados/operations/crush-map/) Weight.  This setting controls the ratio of data that should be distributed to each device.  Right now newly provisioned storage devices on a single node get the same weight.
```

Comment:
the PR for #477 has been merged with https://github.com/rook/rook/commit/41633a9cf039fb22a0d55c7023fa296f9c45db4a.  Feel free to take the next master-latest that has that commit for a test spin, and please update this section to make it more generally applicable to OSD reweight scenarios instead of working around #477.

We could consider moving this section to later in the doc now too because it is now much less important with the fix in.  Thanks @bzub!